### PR TITLE
Add detail about the Commerce.js features used into the readme

### PR DESCRIPTION
**Pull request by ScopeyNZ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,8 +161,47 @@ Because this project is a fully fledged storefront showcasing a custom design an
 - Hosting - [Netlify](https://netlify.com)
 - Styling - [Bootstrap](https://getbootstrap.com) and [SASS](https://sass-lang.com)
 
+## Commerce.js features
+
+This demo store uses a range of features provided by Commerce.js and powered by the Chec dsashboard.
```
Comment (robbieaverill) [inline README.md:166]:
```suggestion
This demo store uses a range of features provided by Commerce.js and powered by the Chec Dashboard.
```

File: README.md [hunk 2]
```diff
@@ -156,8 +161,47 @@ Because this project is a fully fledged storefront showcasing a custom design an
...
+### Customers
+
+Commerce.js provides inbuilt functionality for supporting customer logins without any server side code. This demo store
+features an existing customer login page, and provides detail about previous orders. The customer information is also
+used to pre-populate the checkout with known customer details.
+
+### Payment gateways
+
+This demo store is configured with the Chec "test gateway" out of the box, which provides a handy payment option while
+testing your storefront. Additionally, support for Stripe elements is included if Stripe is configured on the Chec
```
Comment (robbieaverill) [inline README.md:190]:
```suggestion
testing your storefront. Additionally, support for Stripe Elements is included if Stripe is configured on the Chec
```

File: README.md [hunk 3]
```diff
@@ -156,8 +161,47 @@ Because this project is a fully fledged storefront showcasing a custom design an
...
+Dashboard.
+
+#### Enabling Stripe
+
+You must enable Stripe in the Chec dashboard by following the instructions in the dashboard. You may add your sandbox
```
Comment (robbieaverill) [inline README.md:195]:
```suggestion
You must enable Stripe in the Chec Dashboard by following the instructions provided. You may add your sandbox
```

Unsure about the wording here, but using dashboard twice in one sentence feels weird.

Comment (ScopeyNZ) [reply README.md:195]: I like yours better.

**Pull request merged**